Update index.mdx (#500)
primer/brand: apps/docs/content/components/River/index.mdx

## Usage

> Use rivers to showcase features and introduce topics. Rivers are composed of images or videos paired with text content. River's text content includes a heading, body text, and an optional link.

Reviewer: Suggestion for brevity...

```suggestion
Use rivers to showcase features and introduce topics. Rivers are composed of images or videos paired alongside text content like headings, paragraphs, and links.
```

  <Dont>
    <img
      alt=""
      src="https://github.com/primer/brand/assets/6951037/169d5c19-67f4-4f15-8e3b-80c877b34af1"
    />
    <Caption>

>       River component stacks create a unit with coherent flow. Here the last item breaks the flow.

Reviewer: Minor suggestion around active voice...

```suggestion
      Don't break the flow in a stack of rivers that would otherwise stay aligned.
```

  </Dont>
</DoDontContainer>

> ### River Breakout

Reviewer: ```suggestion
### River breakout
```

A river breakout component has an image that fills the width of the container while providing more space for content such as text and lists.

This is useful when you want to create a highlight which can be used to create more emphasis or simply break a monotonous vertical flow of left/right rivers. It can also be helpful on smaller viewports to avoid stacking two images or two chunks of text from adjacent river components. 

> As it's name suggest, a river breakout should not be the first in the stack, it should be in-between two left/right rivers.

Reviewer: Such a good point, I only just realised this 😅 😆

> As it's name suggest, a river breakout should not be the first in the stack, it should be in-between two left/right rivers.

Reviewer: ```suggestion
As its name suggest, a river breakout should not be the first in the stack, it should be positioned in-between two standard rivers.
```

On narrow viewports, there is no left/right, so suggesting that we tweak wording to reference it as the default / standard one.

The image is automatically styled to fit the width of the parent container. If you have bespoke styling requirements, you can disable the provided default styles and style the image manually.

### Additional guidance

> - We recommend using a river left as the first river of the flow to start with a visual and avoid stacking text with above section on mobile

Reviewer: Nit. In RTL reading directions, this would be right-aligned, so could we use directionally-agnostic terms here if possible. E.g. start-aligned if you're referring to 'left'. This is going to become more important as over time, as we expand on i18n.

Could this guidance go into the stacking section too? Feels kind of relevant to that, unless I've misinterpreted it.

Author: I see what you mean by directionally agnostic. I was intending to only talk about the layout and not the text. I don't see any issue with this advice being applied in a RTL language context. Perhaps a different wording could solve this (will make a suggestion)

- River components stacks create a unit, the set should be considered to be the main parent section. The parent section should be spaced as a regular section with the largest spacing while internal elements should use smaller spacing

> - If the image is critical for the user to understand the message, default to right aligned. If the text is critical for the user to understand the image, choose left aligned.

Reviewer: Fine like this for now, but I do think these would make for some **great** visual do/don't examples that can sit under the stacking section above.

Author: Ya, that last section `Additional guidance` feels a bit too abstract and could help some definition via a few visuals and be meshed with the other section. Will do another PR to address this.
